### DISCUSSION TITLE
RSS feeds per page/category/tag

### DISCUSSION THREAD

**Discussion by tombreit** [Guts/mkdocs-rss-plugin#334] [Ideas]
Hi, thanks for your great plugin!

I am looking for a way to create RSS feeds for individual pages: each page could have its own RSS feed. Or dedicated RSS feeds for each category/tag.

Background: users who are only interested in certain topics could receive a feed tailored to their needs.

Is there a way to already do this that I have overlooked?

Best
Thomas

**Comment by Guts** [comment]
This PR by @yrashk should do the job: https://github.com/Guts/mkdocs-rss-plugin/pull/185
